modules/sdk-lib-mpc/test/paillierproof.util.ts

```diff
@@ -1,13 +1,13 @@
 export const mockedPallierProofs: {
```

Comment:
If we're trying to fix spelling everywhere, here's one

Reply:
good catch...